dcrwallet.go

```diff
@@ -137,6 +137,9 @@ func walletMain() error {
 			startPromptPass(w)
 		}
 		startWalletRPCServices(w, rpcs, legacyRPCServer)
+		if w.StakeMiningEnabled {
+			go startTicketPurchase(w)
```

Comment:
I think this will need some additional parameters.

We need the private passphrase because ticket purchasing requires an unlocked wallet.  When not running the wallet with stake mining already enabled (--enablestakemining), it should still be possible to turn on (and off!) the ticket purchaser.  When used from the grpc interface, the passphrase will need to be passed in as a parameter.

It should also pass in some sort of default options here for the "easy" --enablestakemining option.  When started via RPC, more advanced parameters can be used instead of the defaults.